app/controllers/toggle_controller.rb

```diff
@@ -6,7 +6,8 @@ def index
     @facade = OpenStruct.new(
         permanent_toggles: index['permanent'],
         temporary_toggles: index['temporary'],
-        activated_toggles: FeatureFlagState.where(activated: true).map(&:identifier)
+        activated_toggles: FeatureFlagState.where(activated: true).map(&:identifier),
+        updated_timestamps: FeatureFlagState.all.map { |toggle| [toggle.identifier, toggle.updated_at] }.to_h
```

Comment:
Add list of timestamps with their corresponding updated_at value to be used in the view.